core/src/main/java/io/confluent/rest/ApplicationServer.java

```diff
@@ -67,7 +67,7 @@
 // CHECKSTYLE_RULES.OFF: ClassDataAbstractionCoupling
 public final class ApplicationServer<T extends RestConfig> extends Server {
   // CHECKSTYLE_RULES.ON: ClassDataAbstractionCoupling
-  private final T config;
+  private final T serverConfig;
```

Comment:
rename this to make it clear that this config is server config (not specific to connector of listener level).